chunky/src/java/se/llbit/chunky/ui/render/tabs/LightingTab.java

```diff
@@ -94,27 +94,28 @@ public LightingTab() throws IOException {
     sunSamplingStrategy.setTooltip(new Tooltip("Determine how the sun is sampled at each bounce."));
 
     sunIntensity.setName("Sun intensity");
-    sunIntensity.setTooltip("Intensity of the sun on the scene.");
+    sunIntensity.setTooltip("Sunlight intensity modifier.");
     sunIntensity.setRange(Sun.MIN_INTENSITY, Sun.MAX_INTENSITY);
     sunIntensity.makeLogarithmic();
     sunIntensity.clampMin();
     sunIntensity.onValueChange(value -> scene.sun().setIntensity(value));
 
     sunLuminosity.setName("Sun luminosity");
-    sunLuminosity.setTooltip("Absolute brightness of the sun. Used when fast sun sampling is disabled.");
+    sunLuminosity.setTooltip("Absolute brightness of the sun. Only used with Sun Sampling Strategy: OFF or HIGH_QUALITY.");
```

Comment:
```suggestion
    sunLuminosity.setTooltip("Absolute brightness of the sun. Only used with Sun Sampling Strategy is set to OFF or HIGH_QUALITY.");
```

Reply:
I'll make that change, but instead with "Only used *when* Sun Sampling Strategy..."